### PR TITLE
Maybe Fix: PWA top padding

### PR DESCRIPTION
**Pull request by youngkidwarrior**
This might fix. Its hard to test since I can't get login working on my mobile device. It works in iOS simulator

Edit:

SUCCESSS

### CODE REVIEW

**Review by 0xBigBoss — COMMENTED** [review]
hmm to avoid testing in prod,will help test this locally

File: packages/app/utils/usePwa.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const usePwa = () => {
+  if (typeof window !== 'undefined') {
+    return (
+      window.matchMedia('(display-mode: standalone)').matches ||
```
Comment (0xBigBoss) [inline packages/app/utils/usePwa.ts:4]: did u make this up? can u link where u got it from?

**Review by 0xBigBoss — APPROVED** [review]

File: packages/app/utils/usePwa.ts (hunk 1 above)
Comment (youngkidwarrior) [reply packages/app/utils/usePwa.ts:4]: https://stackoverflow.com/questions/41742390/javascript-to-check-if-pwa-or-mobile-web

Comment (0xBigBoss) [reply packages/app/utils/usePwa.ts:4]: thx, i mean document the func lol

Comment (youngkidwarrior) [reply packages/app/utils/usePwa.ts:4]: lmaoooo. sure thing

**Pull request merged**